packages/types/lib/index.ts

```diff
@@ -405,6 +406,11 @@ export interface Rect {
   height: number;
 }
 
+export interface NewWindow {
```

Comment:
is this type unique to Appium or is it standardized somewhere? (could we borrow the type from wdio?)

Reply:
not sure how/where to check this

Reply:
I'm not sure either.  I was just grepping around.  it doesn't look like this response is typed.